### ISSUE TITLE
The example dashboard in the documentation does not work properly

### ISSUE THREAD

**Issue by pablogsal** [buildbot/buildbot#5161]
When trying the example dashboard in the [documentation](http://docs.buildbot.net/latest/manual/customization.html#writing-dashboards-with-flask-or-bottle) I get this error in the inspect tab of my browser:

```
VM450:4 Uncaught ReferenceError: $ is not defined
    at <anonymous>:4:9
    at w (jquery.js:124)
    at qe (jquery.js:5945)
    at M.fn.init.append (jquery.js:6081)
    at M.fn.init.<anonymous> (jquery.js:6175)
    at W (jquery.js:3962)
    at M.fn.init.html (jquery.js:6142)
    at Object.<anonymous> (viewDirective.js:310)
    at angular.js:1390
    at He (angular.js:11269)
```

which comes from this line in the template:

```
        $.getScript("https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.5.0/Chart.bundle.min.js",
``` 

Did something changed in recent versions of buildbot regarding jquery?

Versions:

```
buildbot==2.5.1
buildbot-console-view==2.5.1
buildbot-grid-view==2.5.1
buildbot-waterfall-view==2.5.1
buildbot-worker==2.5.1
buildbot-wsgi-dashboards==2.5.1
buildbot-www==2.5.1
```

**Comment by tardyp** [comment]
Hi, Indeed, it looks like jquery is not available anymore as a global variable. :-/

**Comment by pablogsal** [comment]
> Hi, Indeed, it looks like jquery is not available anymore as a global variable. :-/

Is this intended (and the example should be updated) or considered a bug?

Thanks in advance!

**Comment by tardyp** [comment]
Its a regression on what was available, so a bug..

**Comment by tardyp** [comment]
I made a quick untested fix for you in #5162. Can you try it?
